sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs

```diff
@@ -605,15 +614,15 @@ Page<string> NextPageFunc(string nextLink, int? pageSizeHint)
         /// }
         /// </code>
         /// </example>
-        public virtual AsyncPageable<IncomingRelationship> GetIncomingRelationshipsAsync(string digitalTwinId, CancellationToken cancellationToken = default)
+        public virtual AsyncPageable<IncomingRelationship> GetIncomingRelationshipsAsync(string digitalTwinId, DigitalTwinsListIncomingRelationshipsOptions options = null, CancellationToken cancellationToken = default)
```

Comment:
Consider renaming this type from `DigitalTwinsListIncomingRelationshipsOptions` to `DigitalTwinsGetIncomingRelationshipsOptions`  to stay consistent with the API naming.